### ISSUE TITLE
Linux -  C# extension doesn't install language tools and there's no code completion.

### ISSUE THREAD

**Issue by GlacierFox** [zed-industries/zed#14188]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

After installing the C# extension on Linux I'm not getting anything beyond syntax highlighting, so no code completion.  I may be mistaken but is the extension meant to install Omnisharp to an applicable location? If so, it doesn't seem to be doing anything.

I actually already have omnisharp installed and its on my system path, but I'm under the impression Zed likes to have it installed inside its own .local folder.

### Environment

NAME="Red Hat Enterprise Linux"
VERSION="9.4 (Plow)"
ID="rhel"
ID_LIKE="fedora"
VERSION_ID="9.4"
PLATFORM_ID="platform:el9"

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

[Zed.log](https://github.com/user-attachments/files/16175954/Zed.log)

</pre></details>

**Comment by maxdeviant** [comment]
The extension should be looking for OmniSharp on the $PATH already: https://github.com/zed-industries/zed/blob/37fc4ce09d53d26ba8300ad37b2682cfc1f83836/extensions/csharp/src/csharp.rs#L14-L16

**Comment by GlacierFox** [comment]
> The extension should be looking for OmniSharp on the $PATH already:
> 
> https://github.com/zed-industries/zed/blob/37fc4ce09d53d26ba8300ad37b2682cfc1f83836/extensions/csharp/src/csharp.rs#L14-L16

Yeah I assumed so. I have omnisharp in my path currently. Not sure what's going on with it.

**Comment by notpeter** [comment]
If you're still having trouble getting omnisharp configured please open an issue on the [Zed CSharp Extension](https://github.com/zed-extensions/csharp) repo with as much detail as possible.

Thanks!
